### PR TITLE
Feature/argo doc cleanup

### PR DESCRIPTION
**Pull request by kcoleman731**
This pull request cleans up the ArgoCD docs. A few callouts

1. Leverages variables in commands so developers don't need to copy and paste. 
2. Leverages a forked version of the app of apps repository as the @shapirov103 one uses ssh urls. We will need to move this into quickstart most likely. 

A few items we still need are documented here https://github.com/shapirov103/cdk-eks-blueprint/issues/68

### CODE REVIEW

**Review by shapirov103 — COMMENTED** [review]
A few comments. I would probably invest into automated bootstrapping than into this documentation. We can discuss.

File: lib/addons/argocd/index.ts [hunk 1]
```diff
@@ -6,7 +6,8 @@ export class ArgoCDAddon implements ClusterAddOn {
         clusterInfo.cluster.addHelmChart("argocd-addon", {
             chart: "argo-cd",
             repository: "https://argoproj.github.io/argo-helm",
-            namespace: "argo-cd"
+            version: '3.2.3',
```
[the remark this answers is not in the document]
Comment (kcoleman731) [reply lib/addons/argocd/index.ts:9]: Yeah ... helm version != lib version which isn't great.

File: docs/argo.md [hunk 2]
```diff
@@ -1,108 +1,129 @@
...
 ```
 
+## Deploy workloads to your cluster
 
-## Create Argo Project
 Create a project in Argo by running the following command
+
 ```
-argocd proj create webteam -d https://kubernetes.default.svc,djl-web
+argocd proj create sample -d https://kubernetes.default.svc,djl-web
```
[the remark this answers is not in the document]
Comment (kcoleman731) [reply docs/argo.md:108]: Fixed.

File: docs/argo.md [hunk 3]
```diff
@@ -1,108 +1,129 @@
...
-  GitCommit: b04c25eca8f1660359e325acd4be5338719e59a0
-  GitTreeState: clean
-  GoVersion: go1.14.1
-  Compiler: gc
-  Platform: linux/amd64
-  Ksonnet Version: v0.13.1
-  Kustomize Version: {Version:kustomize/v3.6.1 GitCommit:c97fa946d576eb6ed559f17f2ac43b3b5a8d5dbd BuildDate:2020-05-27T20:47:35Z GoOs:linux GoArch:amd64}
-  Helm Version: version.BuildInfo{Version:"v3.2.0", GitCommit:"e11b7ce3b12db2941e90399e874513fbd24bcb71", GitTreeState:"clean", GoVersion:"go1.13.10"}
-  Kubectl Version: v1.17.8
+kubectl patch $ARGO_SERVER -n argocd -p '{"spec": {"type": "LoadBalancer"}}'
```
[the remark this answers is not in the document]
Comment (kcoleman731) [reply docs/argo.md:32]: Removed in lieu of port forwarding.

File: docs/argo.md [hunk 4]
```diff
@@ -1,108 +1,129 @@
...
 Follow the instructions found [here](https://argoproj.github.io/argo-cd/cli_installation/) as it will include instructions for your specific OS. You can test that the ArgoCD CLI was installed correctly using the following:
+
+```
+argocd version --short --client
+```
+
+You should see output similar to the following:
+
+```
+argocd: v1.8.7+eb3d1fb.dirty
```
[the remark this answers is not in the document]
Comment (kcoleman731) [reply docs/argo.md:16]: Agree. But for a later PR.

[the remark this answers is not in the document]
Comment (shapirov103) [reply docs/argo.md:16]: My point here is that the version that the customer should see must match the version of the helm chart, so let's fix it and state:

```
argocd: v2.0.1+33eaf11.dirty
```

File: docs/argo.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (shapirov103) [reply docs/argo.md]: please remove the djl-web namespace qualifier. It should be https://kubernetes.default.svc,sample (namespace name follows the cluster)

**Review by shapirov103 — COMMENTED** [review]
A couple of minor changes a needed, otherwise lgtm.

File: docs/argo.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kcoleman731) [reply docs/argo.md]: @shapirov103 it has been removed. See the comment is outdated.

**Pull request merged**